src/data/data_classes_utils.py

```diff
@@ -25,7 +25,7 @@
 MASK_DTYPE = np.float32
 TENSOR_DTYPE = torch.float32
 # all relevant edema findings for which masks will be prepared subsequently
-EDEMA_FINDINGS = [k for k in FEATURE_MAP.keys() if k != 'Heart']
+EDEMA_FINDINGS = [k for k in FEATURE_MAP if k != 'Heart' if k != 'Lungs']
 
```

Comment:
@mak-en 

The following approach avoids the need for multiple conditions and makes the code more concise.

`EDEMA_FINDINGS = [k for k in FEATURE_MAP if k not in ['Heart', 'Lungs']]
`